Util-Linux-2.19.1 Gives A GPF During bootstrap

Util-Linux-2.19.1 Gives A GPF during bootstrap. 

Command run:
./rootfs.py -q  -p --build-kernels --update-checksums

Fault Data:

EXCEPTION: General Protection: error code 0x00000000 (0b0)
Process '/usr/libexec/gcc/i386-unknown-linux-musl/4.0.4/collect2' with pid 16296
 cs: 0x0000001b	eip: 0x080587df	efl: 0x00010246	 ss: 0x00000023	esp: 0xbffff418
eax: 0xffffffda	ebx: 0x00000001	ecx: 0xbffff444	edx: 0x00000000
esi: 0x00000008	edi: 0xbffff430	esp: 0xca28bfe4	ebp: 0x00000000
 ds: 0xc9220023	 es: 0xbfff0023	 fs: 0x00000023	 gs: 0x00000023
configure: error: in `/sysa/util-linux-2.19.1/build/util-linux-2.19.1':
configure: error: cannot compute suffix of executables: cannot compile and link
See `config.log' for more details.
Subprocess error 1
ABORTING HARD

WARNING: the last user process has exited. The kernel will stop itself.
kexec_linux: kernel file size: 2360004191
kexec_linux: Invalid kernel signature

**    Safe to Power Off    **
            -or-
** Press Any Key to Reboot **

Fault is reproducible with the given command line. Fault does not occur with a chroot build. I am holding onto the /tmp directory from the last build. if you need anything out of there please let me know.

--- a/sysa/fiwix-1.4.0-lb2/fiwix-1.4.0-lb2.checksums
+++ b/sysa/fiwix-1.4.0-lb2/fiwix-1.4.0-lb2.checksums
@@ -1,1 +1,0 @@
-0de298be5c0b00654ddb226c2e657d4fdd2fa0c613ffb3bb32c23c65d2257bf0  /boot/fiwix

--- a/sysa/fiwix-1.4.0-lb2/sources
+++ b/sysa/fiwix-1.4.0-lb2/sources
@@ -1,1 +1,0 @@
-https://github.com/rick-masters/Fiwix/releases/download/v1.4.0-lb2/fiwix-1.4.0-lb2.tar.gz 44fb14fb666dcf4f5abf7f49eb3f1d92436b9d7353daa23138ff21e9ec78e30f

--- a/sysa/fiwix-1.4.0-lb3/fiwix-1.4.0-lb3.checksums
+++ b/sysa/fiwix-1.4.0-lb3/fiwix-1.4.0-lb3.checksums
@@ -1,0 +1,1 @@
+eda65efb4ca7b1e37cb5e0872e8267251870cdf86b341e6d967c856a926cc3f5  /boot/fiwix

--- a/sysa/fiwix-1.4.0-lb3/fiwix-1.4.0-lb3.kaem
+++ b/sysa/fiwix-1.4.0-lb3/fiwix-1.4.0-lb3.kaem
@@ -18,9 +18,9 @@ cd build
 untar --file ../src/${pkg}.tar
 cd ${pkg}
 
-alias as="tcc -m32 -march=i386 -std=c89 -D__KERNEL__ -DMAX_PID_VALUE=64000000 -DCONFIG_MMAP2 -DCONFIG_64BIT_SYSCALLS -DCONFIG_KEXEC -DNR_PROCS=4096 -DCHILD_MAX=4096 -DOPEN_MAX=1536 -DNR_OPENS=1536 -DINIT_PROGRAM=\"/init\" -DUTS_SYSNAME=\"Linux\" -D__VERSION__=\"tcc\" -traditional -I/sysa/fiwix-1.4.0-lb2/build/fiwix-1.4.0-lb2/include"
+alias as="tcc -m32 -march=i386 -std=c89 -D__KERNEL__ -DMAX_PID_VALUE=64000000 -DCONFIG_MMAP2 -DNO_CONFIG_OFFSET64 -DCONFIG_64BIT_SYSCALLS -DCONFIG_KEXEC -DNR_PROCS=4096 -DCHILD_MAX=4096 -DOPEN_MAX=1536 -DNR_OPENS=1536 -DINIT_PROGRAM=\"/init\" -DUTS_SYSNAME=\"Linux\" -D__VERSION__=\"tcc\" -traditional -I/sysa/${pkg}/build/${pkg}/include"
 
-alias cc="tcc -m32 -march=i386 -std=c89 -D__KERNEL__ -DMAX_PID_VALUE=64000000 -DCONFIG_MMAP2 -DCONFIG_64BIT_SYSCALLS -DCONFIG_KEXEC -DNR_PROCS=4096 -DCHILD_MAX=4096 -DOPEN_MAX=1536 -DNR_OPENS=1536 -DINIT_PROGRAM=\"/init\" -DUTS_SYSNAME=\"Linux\" -D__VERSION__=\"tcc\" -I/sysa/fiwix-1.4.0-lb2/build/fiwix-1.4.0-lb2/include -O2 -fno-pie -fno-common -ffreestanding -Wall -Wstrict-prototypes"
+alias cc="tcc -m32 -march=i386 -std=c89 -D__KERNEL__ -DMAX_PID_VALUE=64000000 -DCONFIG_MMAP2 -DNO_CONFIG_OFFSET64 -DCONFIG_64BIT_SYSCALLS -DCONFIG_KEXEC -DNR_PROCS=4096 -DCHILD_MAX=4096 -DOPEN_MAX=1536 -DNR_OPENS=1536 -DINIT_PROGRAM=\"/init\" -DUTS_SYSNAME=\"Linux\" -D__VERSION__=\"tcc\" -I/sysa/${pkg}/build/${pkg}/include -O2 -fno-pie -fno-common -ffreestanding -Wall -Wstrict-prototypes"
 
 cd kernel
 as -c -o boot.o boot.S

--- a/sysa/fiwix-1.4.0-lb3/sources
+++ b/sysa/fiwix-1.4.0-lb3/sources
@@ -1,0 +1,1 @@
+https://github.com/rick-masters/Fiwix/releases/download/v1.4.0-lb3/fiwix-1.4.0-lb3.tar.gz 525a24e32571e574da76e2a8c898f4eab371dfb85302d545c498a21c8028283d

--- a/sysa/run.kaem
+++ b/sysa/run.kaem
@@ -57,7 +57,7 @@ fi
 
 if match x${BUILD_FIWIX} xTrue; then
     # The Fiwix kernel
-    pkg="fiwix-1.4.0-lb2"
+    pkg="fiwix-1.4.0-lb3"
     cd ${pkg}
     kaem --verbose --file ${pkg}.kaem
     cd ..

